### PR TITLE
openssh|systemd: depend on libxcrypt

### PR DESCRIPTION
**Pull request by SMillerDev**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Fixes the issues found in #98809

### CODE REVIEW

**Comment by bayandin** [comment]
> chown: /opt/homebrew/Cellar/mailutils/3.15/bin/dotlock: Operation not permitted
>   chmod: Unable to change file mode on /opt/homebrew/Cellar/mailutils/3.15/bin/dotlock: Operation not permitted

A similar problem was fixed in https://github.com/Homebrew/brew/pull/13404, probably it requires tuning the sandbox again

**Comment by carlocab** [comment]
> ```
> chown root
> ```

might be the problem. Not sure why it didn't come up before though.

**Comment by Bo98** [comment]
Doesn't seem sandbox related chmod also fails outside of any sandbox.

**Comment by mistydemeo** [comment]
I can repro outside of CI. `brew install -s mailutils` exhibits the issue.

From reviewing the makefiles, it does appear that the `chown root:mail` is hardcoded - it's not that it's dynamically determining which user to `chown` to.

**Comment by Bo98** [comment]
Posted the response in the other issue.

In terms of this PR: these should really have revision bumps since we will (eventually) be removing libcrypt.so.1 from our glibc formula.

File: Formula/mailutils.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Mailutils < Formula
   mirror "https://ftpmirror.gnu.org/mailutils/mailutils-3.15.tar.gz"
   sha256 "91c221eb989e576ca78df05f69bf900dd029da222efb631cb86c6895a2b5a0dd"
   license "GPL-3.0-or-later"
+  revision 1
```
Comment (carlocab) [inline Formula/mailutils.rb:8]: Can't merge without breaking the `mailutils` bottle on Monterey ARM64 unfortunately.

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**